modules/custom/ginvite/ginvite.install

```diff
@@ -40,3 +40,17 @@ function ginvite_update_8001() :void {
     }
   }
 }
+
+/**
+ * Update exposed filter button name.
```

Comment:
We changed ginvite from custom to community module, I'm not sure we should modify the community module. Can we remove this settings?

Reply:
@tbsiqueira done